modules/ting/src/Search/TingSearchSort.php

```diff
@@ -20,6 +20,7 @@ class TingSearchSort {
   // TODO: Could be an enum.
   const DIRECTION_ASCENDING = 'asc';
   const DIRECTION_DESCENDING = 'desc';
+  const DIRECTION_NONE = '_none';
```

Comment:
Direction none is an odd concept. Perhaps adding a comment about it would be good.